### ISSUE TITLE
Document TF Migrations then Wrap Up TF Issues and PRs

### ISSUE THREAD

**Issue by martinpeck** [opencb/opencga#1208]
We need to...

-[ ] Document where we are with the Terraform migration within an Issue
-[ ] Create branch to hold the TF work
-[ ] Remove the TF files/work from the `azure` branch
-[ ] Close/remove the issue from the `azure` project

This will allow others to see the work we did with Terraform, should it be needed/useful in the future,  without it remaining in the `azure` branch.

**Comment by martinpeck** [comment]
A branch that represents a snapshot of the `azure` branch has been created:

https://github.com/opencb/opencga/tree/azure-tf

**Comment by martinpeck** [comment]
The following issues will be removed from the `azure` project, and then closed:

1. #1092 Create ARM Based Deployment "Stub" that kicks off Terraform Deployment of OpenCGA 
2. #1124 TF Migrate: Log Analytics
3. #1122 TF Migrate: Solr/Zookeeper
4. #1125 TF Migrate: App Gateway
5. #1126 TF Migrate: TShirt Sizing
6. #1123 TF Migrate: Avere
7. #1127 TF Migrate: Integration

The following **PRs** will be removed from the `azure` project, and then closed:

1. #1129 WIP: TF Migrate log analytics
2. #1128 WIP: Add NSGs to subnets using Terraform

**Comment by martinpeck** [comment]
Issues closed.
PRs closed.

**Comment by martinpeck** [comment]
Awaiting review of PR that removes the `/terraform/` folder from the repo

**Comment by lawrencegripper** [comment]
That PR looks good so I've merged and updated the checklist on this one.

**Comment by martinpeck** [comment]
@lawrencegripper so, this just leaves the "document where we are" task. Are you OK to pick that up (as I think you wanted to write up the approach)?

**Comment by lawrencegripper** [comment]
Yes, I'll add something to this issue with some details or think it's best to add a README to the `azure-tf` branch?

**Comment by martinpeck** [comment]
suggest add a README to the `azure-ft` branch (so that people can amend it or comment on it easier)
